docs/UserGuide.adoc

```diff
@@ -151,6 +151,11 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+== Sorting the list of persons : `sort`
+
+Sorts lexicographically and lists all persons, along with their non-private details, in the address book. +
```

Comment:
Maybe you can abstract further? Instead of making this method sort AND list, maybe make it sort only, to obey SLAP (Single Level of Responsibility Principle)